tamr_unify_client/client.py

```diff
@@ -81,25 +83,41 @@ def request(self, method, endpoint, **kwargs):
 
         return response
 
-    def get(self, endpoint, **kwargs):
-        """Calls :func:`~tamr_unify_client.Client.request` with the ``"GET"`` method.
+    def request_json(self, method, endpoint, **kwargs):
```

Comment:
An alternative (and, maybe, cleaner?) approach would be to keep `get()`, but override the `.json()` method of the returned object to call `self.raise_for_status()` before delegating to the default `.json()`.

Reply:
It's too bad that `raise_for_status()` doesn't return `self`, so you can't do `client.get(...).raise_for_status().json()`, which would be ideal. BUT, we can inject a method that does `raise_for_status()` then returns `self`. I think that's even better than magical mystery `get()` - add a `successful()` method that does `raise_for_status` and returns `self`, then change call sites to `client.get(...).successful().json()`